src/newSeed/baseStage.scss

```diff
@@ -87,6 +121,9 @@
           .deleteLink {
             cursor: pointer;
             justify-self: flex-end;
+            i {
+              font-family: "Font Awesome\ 5 Free";
```

Comment:
setting this font-family should not be necessary.  what does it accomplish here?